src/Cli/dotnet/CommandFactory/CommandResolution/PathCommandResolverPolicy.cs

```diff
@@ -16,7 +16,7 @@ public static CompositeCommandResolver Create()
     {
         var environment = new EnvironmentProvider();
 
-        var platformCommandSpecFactory = default(IPlatformCommandSpecFactory);
+        IPlatformCommandSpecFactory platformCommandSpecFactory;
```

Comment:
I'd like this better as a ternary

Reply:
I'll change it. Note that our (VS) tooling can't do that automatically, but Resharper can. Meaning, it can detect when a ternary can be used instead.